Tests/GitHubLabels.tests.ps1

```diff
@@ -272,7 +272,7 @@ if ($script:accessTokenConfigured)
         $null = Remove-GitHubRepository -OwnerName $script:ownerName -RepositoryName $repositoryName
     }
 
-    Describe 'Adding and removing labels to an issue'{
+    Describe 'Adding labels to an issue'{
         $repositoryName = [Guid]::NewGuid().Guid
```

Comment:
I have the usual feedback about leaving `$repositoryName` as just `[Guid]::NewGuid()`

Reply:
I think you are looking at an older revision.